src/log-analytics-solution/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '1.0.0'
+VERSION = '1.0.1'
 
```

Comment:
Please update package name in setup from `log_analytics_solution` to `log-analytics-solution`

Reply:
refer to this pr: https://github.com/Azure/azure-cli-extensions/pull/7131